rdrf/rdrf/forms/dynamic/fields.py

```diff
@@ -44,17 +45,23 @@ def validate(self, value):
 
 class FileTypeRestrictedFileField(FileField):
 
+    def _find_whitelisted_file_types(self, file_extension):
+        return WhitelistedFileExtension.objects.filter(file_extension__iexact=file_extension)
+
     def _find_blacklisted_mime_type(self, mt):
         from rdrf.models.definition.models import BlacklistedMimeType
         return BlacklistedMimeType.objects.filter(mime_type=mt).first()
 
     def validate(self, value):
         if not value:
             return super().validate(value)
-        __, ext = os.path.splitext(value._name)
+        __, ext = os.path.splitext(value.name)
```

Comment:
Good catch. Not sure why it used the underscore value initially, might've just missed the `name` property.